### ISSUE TITLE
USING LINK IN NEXTJS 

### ISSUE THREAD

**Issue by ErnestDaDev** [Abusayid693/react-hamburger-menus#44]
Nice package. There seems to be an issue with the menu closing on mobile devices. I have the responsive menu in a global layout using NextJS so when it switches to mobile and you tap a menu item, it does not close up the menu.

**Comment by Abusayid693** [comment]
@ErnestDaDev Thanks for reporting, will fix it ASAP

**Comment by Abusayid693** [comment]
@ErnestDaDev This is not a bug wrap Navigation component on each page or route instead of global layout.

**Comment by ErnestDaDev** [comment]
> @ErnestDaDev This is not a bug wrap Navigation component on each page or route instead of global layout.

Yeah. That fixes it. But check after deploying to the cloud the disadvantage is when the page is loading, it displays a blank screen, which of course can be replaced with a loader but it's overkill simply because of a navbar. You can provide an option to provide a state to handle the toggle and animate down of the menu items.

**Comment by Abusayid693** [comment]
> > @ErnestDaDev This is not a bug wrap Navigation component on each page or route instead of global layout.
> 
> Yeah. That fixes it. But check after deploying to the cloud the disadvantage is when the page is loading, it displays a blank screen, which of course can be replaced with a loader but it's overkill simply because of a navbar. You can provide an option to provide a state to handle the toggle and animate down of the menu items. ![error](https://user-images.githubusercontent.com/78008731/173193541-a6c9f0de-90e6-4235-9817-2409f8a641c7.gif) [ ](https://user-images.githubusercontent.com/78008731/173193541-a6c9f0de-90e6-4235-9817-2409f8a641c7.gif) ![error](https://user-images.githubusercontent.com/78008731/173193541-a6c9f0de-90e6-4235-9817-2409f8a641c7.gif) [ ](https://user-images.githubusercontent.com/78008731/173193541-a6c9f0de-90e6-4235-9817-2409f8a641c7.gif)

Yes this seems like problematic from user experience perspective, I will add a prop to handle the state manually, feel free to open any PR for this fix

**Comment by ErnestDaDev** [comment]
> > Yeah. That fixes it. But check after deploying to the cloud the disadvantage is when the page is loading, it displays a blank ...
> 
> Yes this seems like problematic from user experience perspective, I will add a prop to handle the state manually, feel free to open any PR for this fix

Yes, created one. Kindly check it out.
